Clue/Classes/Protocols/CLUInfoModule.h

```diff
@@ -19,12 +20,12 @@
 @required
 
 /**
- Initialize info module with specific writer which implements `CLUWritable` protocol. So module will be able to record/write required information.
+ Initialize info module with specific `JSONWriter` instance. So module will be able to record/write required information.
  
- @param writer Writer object which implements `CLUWritable` protocol. Responsible for actual writing information to some specific file
+ @param writer `JSONWriter` instance responsible for actual writing information to some specific file
  @return New instance of info module
  */
-- (instancetype)initWithWriter:(id <CLUWritable>)writer;
+- (instancetype)initWithWriter:(JSONWriter *)writer;
```

Comment:
I didn't touch `CLURecordableModule` because I wanted to restrict the changes in this PR to writing JSON data.

On second thought, though, making these changes to `CLUInfoModule.h` is not needed. I will revert them, and re-introduce `id <CLUWritable>`, in the next PR.